packages/admin-panel/src/routes.js

```diff
@@ -200,4 +201,16 @@ export const ROUTES = [
       },
     ],
   },
+  {
+    label: 'External Data',
```

Comment:
It will be nice to add DHIS2 Instances to this tab once we've got them in the Admin Panel @IgorNadj

Reply:
Yeah I was waiting until we consolidated the concept of data services before adding them to the admin panel. Right now it's kind of messy, we would have a tab for each type. Maybe we'll pop these db connections in as data services at that point too.